orbotservice/src/main/java/org/torproject/android/service/vpn/OrbotVpnManager.java

```diff
@@ -446,23 +380,20 @@ private void startDNS (String pdnsPath, String torDnsHost, int torDnsPort, Strin
 
 	File filePdnsPid;
 
-    private boolean stopDns ()
-	{
+    private boolean stopDns () {
 		if (filePdnsPid != null && filePdnsPid.exists()) {
 			List<String> lines = null;
 			try {
 				lines = IOUtils.readLines(new FileReader(filePdnsPid));
 				String dnsPid = lines.get(0);
-				killProcess(dnsPid, "");
```

Comment:
I think not using `import static` on utility methods like this can really improve code readability